modules/messages/server/controllers/messages.server.controller.js

```diff
@@ -280,6 +281,16 @@ exports.send = function(req, res) {
       });
     },
 
+    // Here we collect some message data into the influxdb
+    function (message, done) {
+      // the module returns a Promise
+      // it is async, but we'll let it run wild (not waiting for the promise to
+      // be fulfilled or rejected)
+      messageToInfluxService(message);
```

Comment:
Not sure what this comment means. Are we saying that we don't care if the promise completes or not? Would be great to make it a bit clearer. I'm a big fan of humour, so long as it's clear what the meaning is! :-)